src/test/groovy/com/netflix/gradle/plugins/deb/DebPluginTest.groovy

```diff
@@ -67,6 +67,33 @@ class DebPluginTest extends ProjectSpec {
         ['testpkg_1.0.0-1_all.deb', 'testpkg_1.0.0-1_all.changes'].toSet() == declaredOutputs
     }
 
+    def 'createDirectoryEntry generates postinst install line'() {
+        given:
+        String testDir = "${project.buildDir.path}/testdir"
+        File srcDir = new File(testDir, 'a')
+        srcDir.mkdirs()
+
+        when:
```

Comment:
NITPICK: I would tend to have the when block only do `debTask.execute()` and have the rest in given